functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -258,22 +344,54 @@ Then it will export the results to Export-DbaDiagnosticQuery.
                                 DatabaseSpecific = $scriptpart.DBSpecific
                                 DatabaseName     = $null
                                 Notes            = $null
-                                Result           = $result | Select-Object * -ExcludeProperty RowError, RowState, Table, ItemArray, HasErrors
+                                Result           = Select-DefaultView -InputObject $result -Property *
                             }
+
+                        }
+                    }
+                    else {
+                        # if running WhatIf, then return the queries that would be run as an object, not just whatif output
+
+                        [pscustomobject]@{
+                            ComputerName     = $server.NetName
+                            InstanceName     = $server.ServiceName
+                            SqlInstance      = $server.DomainInstanceName
+                            Number           = $scriptpart.QueryNr
+                            Name             = $scriptpart.QueryName
+                            Description      = $scriptpart.Description
+                            DatabaseSpecific = $scriptpart.DBSpecific
+                            DatabaseName     = $null
```

Comment:
Change this to just `Database`

Reply:
Change other occurrences of the `DatabaseName` property to `Database` as well for consistency.